example/components/linux.py

```diff
@@ -0,0 +1,135 @@
+"""Giraffe component is the API to Giraffe component."""
+from typing import Any, Optional, TextIO, List, NamedTuple
+
+import asyncio
+import click
+import colored
+import contextlib
+import time
+import os
+
+from Octavius.lego.components import RPyCComponent
+
+
+class LinuxRPyCComponent(RPyCComponent):
+    """An extended interface for RPyC component from linux type."""
+
+    def __init__(self, *args, **kwargs):
+        """Initializes a linux RPyC component."""
+
+        super().__init__(*args, **kwargs)
+
+        self._unallowed_logs: List[str] = list()
+        self._allowed_logs: List[str] = list()
+        self._expected_logs: List[str] = list()
+
+    @property
+    def unallowed_logs(self) -> List[str]:
+        """Gets the unallowed logs."""
+
+        return self._unallowed_logs
+
+    @property
+    def allowed_logs(self) -> List[str]:
+        """Gets the allowed logs."""
+
+        return self._unallowed_logs
+
+    @property
+    def expected_logs(self) -> List[str]:
+        """Gets the expected logs."""
+
+        return self._unallowed_logs
+
+    @contextlib.contextmanager
+    def monitor_logs(self, path: str) -> Any:
+        """Monitors on file or directory in this remove machine.
```

Comment:
```suggestion
        """Monitors a file or directory on the remote machine.
```
Please stop copying this line without fixing it!

Reply:
Look below